src/clarity-angular/layout/tabs/aria-service.ts

```diff
@@ -0,0 +1,7 @@
+import {Injectable} from "@angular/core";
+
+@Injectable()
+export class AriaService {
```

Comment:
Hmm that's a good point. I'm not sure of other components that will use them yet. `aria-controls` and `aria-labelledby` are pretty limited in use cases since they depend on each other and only certain components like tabs would have them. I'm open to moving them though, if we find another component that could use an `aria-` related service. I'll add a TODO with a note to consider moving this if we run into another case where this service is useful.